### DISCUSSION TITLE
Create empty (0 sat) LND accounts?

### DISCUSSION THREAD

**Discussion by jimmydjabali** [lightningnetwork/lnd#8441] [Q&A]
Hello,

I'd like to create empty accounts with litd but seems the minimum to create an account is 1 sat.
Is this is a technical limitation why it's not possible to create an account with an initial balance of 0 sat, or is there a workaround to make it possible?

Thanks a lot!

**Comment by ellemouton** [comment]
Hi @jimmydjabali  

The error is coming from [this line](https://github.com/lightninglabs/lightning-terminal/blob/c03b040e4de02c813d7613f580df0e6f7ae5f111/cmd/litcli/accounts.go#L117-L119) which is only on the CLI level. So you should be able to get around this by using the RPC directly if you need to. But yeah - I think there is no reason that it cannot be 0 but I stand to be corrected (cc @guggero )

(btw - i think this discussion should have been created on the [LiTd repo](https://github.com/lightninglabs/lightning-terminal) instead)

**Reply by jimmydjabali** [reply]
Unfortunately, there is a similar error `a new account cannot have balance of 0` thrown here:
https://github.com/lightninglabs/lightning-terminal/blob/master/accounts/store.go#L111

**Comment by ellemouton** [comment]
@jimmydjabali  - will be fixed in : https://github.com/lightninglabs/lightning-terminal/pull/718
